scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -223,12 +224,12 @@ export function RelatedItemsDirective(
              * @param {String} fieldId
              * @return {int} nextKey
              */
-            function getNextKey(associations, fieldId) {
+            function getNextKeyAndOrder(associations, fieldId) {
                 for (let i = 1; ; i++) {
                     const key = fieldId + '--' + i;
 
                     if (associations[key] == null) {
-                        return key;
+                        return {key, order: i};
```

Comment:
I don't see the order field used anywhere. Is it only meant to be used on the back-end?

Reply:
I missed it.